src/Marten.Testing/ConnectionSource.cs

```diff
@@ -4,8 +4,9 @@ namespace Marten.Testing
 {
     public class ConnectionSource : ConnectionFactory
     {
-        public static readonly string ConnectionString = Environment.GetEnvironmentVariable("marten_testing_database")
-            ?? "Host=localhost;Port=5432;Database=postgres;Username=postgres;password=postgres";
+        //public static readonly string ConnectionString = Environment.GetEnvironmentVariable("marten_testing_database")
```

Comment:
@cocowalla please revert this leftover

Reply:
Hah, I did the same thing last time I sent a Marten PR (too lazy to restart my IDE after settling an environment variable), and this time I told myself I'd definitely change it back before pushing the code!